app/services/doc_auth/lexis_nexis/error_generator.rb

```diff
@@ -0,0 +1,80 @@
+module DocAuth
+  module LexisNexis
+    class UnknownTrueIDError < StandardError; end
+
+    class ErrorGenerator
+      # These constants are the key names for the TrueID errors hash that is returned
+      ID = 'id'.to_sym
+      FRONT = 'front'.to_sym
+      BACK = 'back'.to_sym
+
+      TRUE_ID_MESSAGES = {
+        '1D Control Number Valid': { type: BACK, message: I18n.t('doc_auth.errors.lexis_nexis.1d_control_number_check')},
```

Comment:
assigning an `I18n.t` lookup to a constant is bad because if the locale changes, the value won't, I would recommend putting the key in the constant and I18n it later

Reply:
Good call. I hadn't thought of that. I fixed in local and will update when I have fixed some of the failing stuff.